### ISSUE TITLE
`yarn start` exits with `ok` in less than 4 seconds

### ISSUE THREAD

**Issue by andersonmmi** [makerdao/community-portal#207]
While trying to run the development server locally, gatsby fails quietly. I was able to run the development server through Vagrant, but it fails on my native environment.

I ran `yarn clean && yarn start` with the verbose flag and this is the output:
```
aaronlynanderson@Environment-2 community-portal % yarn clean --verbose
yarn run v1.22.4
$ gatsby clean --verbose
verbose set gatsby_log_level: "verbose"
verbose set gatsby_executing_command: "clean"
verbose loading local command from: /Users/aaronlynanderson/Documents/MakerDAO/community-portal/node_modules/gatsby/dist/commands/clean.js
verbose running command: clean
info Deleting .cache, public
info Successfully deleted directories
✨  Done in 8.15s.
aaronlynanderson@Environment-2 community-portal % yarn start --verbose
yarn run v1.22.4
$ npm run acrollout && npm run develop --verbose
npm info it worked if it ends with ok
npm verb cli [
npm verb cli   '/Users/aaronlynanderson/.nvm/versions/node/v14.5.0/bin/node',
npm verb cli   '/Users/aaronlynanderson/.nvm/versions/node/v14.5.0/bin/npm',
npm verb cli   'run',
npm verb cli   'develop',
npm verb cli   '--verbose'
npm verb cli ]
npm info using npm@6.14.5
npm info using node@v14.5.0
npm verb run-script [ 'predevelop', 'develop', 'postdevelop' ]
npm info lifecycle community-development-portal@0.1.0~predevelop: community-development-portal@0.1.0
npm info lifecycle community-development-portal@0.1.0~develop: community-development-portal@0.1.0
npm info lifecycle community-development-portal@0.1.0~develop: ignored because ignore-scripts is set to true community-development-portal@0.1.0
npm info lifecycle community-development-portal@0.1.0~postdevelop: community-development-portal@0.1.0
npm verb exit [ 0, true ]
npm timing npm Completed in 346ms
npm info ok 
✨  Done in 2.55s.
```

**Comment by andersonmmi** [comment]
@MaximumCrash any ideas?

**Comment by MaximumCrash** [comment]
I'll changing the package.json for `npm run start` to use `gatsby develop` instead of `npm run develop`

**Comment by MaximumCrash** [comment]
Closed by #217
